velox/exec/SortWindowBuild.cpp

```diff
@@ -217,12 +218,8 @@ void SortWindowBuild::sortPartitions() {
   RowContainerIterator iter;
   data_->listRows(&iter, numRows_, sortedRows_.data());
 
-  std::sort(
-      sortedRows_.begin(),
-      sortedRows_.end(),
-      [this](const char* leftRow, const char* rightRow) {
-        return compareRowsWithKeys(leftRow, rightRow, allKeyInfo_);
```

Comment:
Can we remove compareRowsWithKeys? Is it still used by sort window build? Thanks!

Reply:
compareRowsWithKeys is required to demarcate partitions after the entire input is sorted by (partition keys + order by keys). So it cannot be removed.